docs/user-guide/docker-plugin-drivers.md

```diff
@@ -3,31 +3,16 @@ title: Overview
 
 ---
 ## Overview
-The Docker volume plugin supports the below platforms and the corresponding drivers for those. The plugin supports all volume provisioning and managenment operations, defined by the Docker Volume plugin interface, on both platforms. Both drivers allow users to provision and use VMDK backed volumes for containers in Docker.
-
-1. vSphere
-2. Photon
-
-<script type="text/javascript" src="https://asciinema.org/a/80417.js" id="asciicast-80417" async></script>
+The Docker volume plugin supports the vSphere platform and provides corresponding driver for it. The plugin supports all volume provisioning and managenment operations, defined by the Docker Volume plugin interface. The corresponding driver allow users to provision and use VMDK backed volumes for containers in Docker.
```

Comment:
nit: "The corresponding driver allow" => "The corresponding driver allows"

Reply:
Done.